server/models/book.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose');
+
+const bookSchema = mongoose.Schema({
+    name:{
+        type:String,
+        required:true
+    },
+    author:{
+        type:String,
+        required:true
+    },
+    review:{
+        type:String,
+        required:true
+    },
+    pages:{
+        type:String,
+        default:'n/a'
+    },
+    rating:{
+        type:Number,
+        required:true,
+        min:1,
+        max:5
+    },
+    price:{
+        type:String,
+        default:'n/a'
+    },
+    ownerID:{
+        type:String,
+        required:true
+    }
```

Comment:
Are you planning to generate an ID? Mongo does generate a random ID for each document stored you might need to have a look at it otherwise feel free to ignore this comment.

Reply:
Mongo is generate one, will remove it